### DISCUSSION TITLE
第1节课笔记(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#54] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by myaijarvis** [comment]
【参考：[【笔记】书生·浦语大模型实战营——第一课-CSDN博客](https://blog.csdn.net/Jruo911/article/details/135373499 )】

**Reply by mattheliu** [reply]
佬，你这个是第二节课的把 :flushed:

**Comment by yestolife** [comment]
[第一课笔记，书生浦语大模型全链路开源体系](http://t.csdnimg.cn/7qTKh)

**Reply by mattheliu** [reply]
佬下次可以加上图片哦 :blush:

**Comment by DrZhang1208** [comment]
[书生·浦语大模型全链路开源开放体系](https://blog.csdn.net/qq_42866033/article/details/135374144?spm=1001.2014.3001.5501)

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/vhp2a5369v2fk0mx?singleDoc# 《书生·浦语大模型全链路开源体系》

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/ocli6hn45av7gbe4?singleDoc# 《lesson1 浦语大模型全链路开源开放体系笔记》

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/1 书生·浦语大模型全链路开源体系

**Comment by Heitao5200** [comment]
https://github.com/Heitao5200/my_blog/blob/master/docs/组队学习/大模型实战/TASK1.md 书生·浦语大模型全链路开源体系

**Comment by zxp22** [comment]
https://www.zhihu.com/pin/1726901747002642432 第一节课笔记

**Comment by ZnRbCl3** [comment]
【第一讲 书生•浦语大模型全链路开源体系 -  CSDN App】http://t.csdnimg.cn/Wfjvc

**Comment by lengbaihang** [comment]
书生浦语大模型全链路开源体系  https://blog.csdn.net/lengbaihang/article/details/135433745?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135433745%22%2C%22source%22%3A%22lengbaihang%22%7D[](url)[](url)

**Comment by tackhwa** [comment]
[第1节课笔记，书生·浦语大模型全链路开源体系](https://nutritious-chill-7a9.notion.site/ca4f82d6bde74e5282162480ff60c150?pvs=4)

**Comment by xwhclaire** [comment]
[Lecture 1 Notes 书生·浦语大模型全链路开源体系](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/%23%20Lecture%201%20%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E4%BD%93%E7%B3%BB.md)

**Comment by eighteen-wu** [comment]
[书生·浦语大模型全链路开源体系笔记](https://blog.csdn.net/weixin_43882408/article/details/135439395?spm=1001.2014.3001.5502)

**Comment by JinFish** [comment]
[第一节课笔记](https://blog.csdn.net/wildland/article/details/135442484?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135442484%22%2C%22source%22%3A%22wildland%22%7D)

**Comment by caohuikang** [comment]
第一节课笔记：
1、书生·浦语大模型系列包括轻量级InternLM-7B、重量级InternLM-20B、重量级InternLM-123B三种。
2、书生·浦语开放体系包括数据、预训练、微调、部署、评测、应用

**Comment by haokun-dev** [comment]
第一节课笔记【【书生·浦语大模型实战营】书生·浦语大模型全链路开源体系 -  CSDN App】http://t.csdnimg.cn/8Qh4u

**Comment by FEDxy** [comment]
Lesson.1_书生•浦语大模型_全链路开源开放体系笔记

https://zhuanlan.zhihu.com/p/676520683?

**Comment by ggfengxuan** [comment]
[1.书生·浦语大模型全链路开源开放体系笔记](https://qa5023amair.feishu.cn/wiki/TmbKwc1sziQhcHkyoHgcm8rrnbh?from=from_copylink)

**Comment by ZiHAO-LI-cmd** [comment]
[书生·浦语大模型实战营 Lesson 1](http://t.csdnimg.cn/dh4uY)

**Comment by D-JI-Feng** [comment]
笔记写在飞书云文档https://k0ilmnk4dz3.feishu.cn/wiki/YkWOwMLcfi9U9VkVNjEcK8ksnQM?from=from_copylink

**Comment by Noah-Lan** [comment]
[第一节笔记](https://github.com/Noah-Lan/Document-Transfer-Sation/blob/main/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E4%BD%93%E7%B3%BB.pdf)

**Comment by scorpionfree98** [comment]
[第一节课笔记](http://t.csdnimg.cn/PVFw6)
